rest_framework/serializers.py

```diff
@@ -502,7 +502,7 @@ def get_value(self, dictionary):
         # We override the default field access in order to support
         # lists in HTML forms.
         if html.is_html_input(dictionary):
-            return html.parse_html_list(dictionary, prefix=self.field_name)
+            return html.parse_html_list(dictionary, prefix=self.field_name) or empty
```

Comment:
Not sure about including this on `ListField` and `ListSerializer` - in those cases wouldn't we most likely want the empty list to be default if nothing is present?